src/lib/media-query/match-media.ts

```diff
@@ -5,42 +5,83 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import {
-  Inject,
-  Injectable,
-  NgZone,
-  PLATFORM_ID,
-  RendererFactory2,
-  RendererType2,
-  ViewEncapsulation,
-} from '@angular/core';
+import {Inject, Injectable, NgZone, PLATFORM_ID} from '@angular/core';
 import {DOCUMENT, isPlatformBrowser} from '@angular/common';
 import {BehaviorSubject} from 'rxjs/BehaviorSubject';
 import {Observable} from 'rxjs/Observable';
 import {filter} from 'rxjs/operators/filter';
 
 import {MediaChange} from './media-change';
+import {BreakPoint} from '../media-query/breakpoints/break-point';
 
 /**
- * EventHandler callback with the mediaQuery [range] activates or deactivates
+ * Special server-only class to simulate a MediaQueryList and
+ * - supports manual activation to simulate mediaQuery matching
+ * - manages listeners
  */
-export interface MediaQueryListListener {
-  // Function with Window's MediaQueryList argument
-  (mql: MediaQueryList): void;
-}
+export class ServerMediaQueryList implements MediaQueryList {
```

Comment:
This should be in a separate file
(generally all server-specific stuff should be in separate files so that you can better ensure you're only getting what you need on the client)

Reply:
Added to a separate file with a separate implementation of `MatchMedia` for the server as suggested below